message.d.ts

```diff
@@ -661,6 +666,8 @@ export interface Sticker {
   premium_animation?: File;
   /** For mask stickers, the position where the mask should be placed */
   mask_position?: MaskPosition;
+  /** For custom emoji stickers, unique identifier of the custom emoji */
+  custom_emoji_id?: string;
```

Comment:
In 6.2 changelogs, they have added `type` property. (Seems like you've added it in the `StickerSet` but not here)

Reply:
Yeah this is my mistake because I'm not careful in reading the documentation

Reply:
Its okay, don't worry :) You have time to fix things.